index.html

```diff
@@ -93,8 +93,8 @@ <h1>Persistent Data (undertale.ini)</h1>
             </button>
           </div>
 
-          <div class="grid-cols-3">
-            <label class="mt-1.5">Name</label>
+          <div class="grid-cols-3 mt-1.5">
+            <label>Name</label>
```

Comment:
If you want to tackle it here, these labels are supposed to have a `for` attribute pointing to the `<input>` element they represent, so clicking the label delegates that event to the input element. For example, here it would be `for="ini-name"`.